lib/full_text_search/searcher.rb

```diff
@@ -226,6 +226,7 @@ def output_columns
         "content_snippets",
         "id",
         "last_modified_at",
+        "registered_at",
```

Comment:
How about showing this too in search result list?

Reply:
@kou 
May I ask you the use-case about it? I guess we will use it the following place?

https://github.com/clear-code/redmine_full_text_search/blob/3c14ef3c50357f8df2764ccc3a3fd02267b2cbe6/app/views/search/index.html.erb#L182

---

Do you mean we will include `record.registered_at` in the following search result list?
If so, I'm thinking about how I should return it.

- Return the different fields
  - Add the following fields to the response fields
      -  `api.last_modified_at: record.registered_at`
      - `api.registered_at: record.registered_at`
- Return it as the `api.datetime`
  - Change the value depending on the `order_target` field although I haven't had any image to implement it.
    - if order_target is registered_at, `api.datetime: record.registered_at` 
    - if order_target is last_modified_at, `api.datetime: record.last_modified_at` 

https://github.com/clear-code/redmine_full_text_search/blob/3c14ef3c50357f8df2764ccc3a3fd02267b2cbe6/app/views/search/index.api.rsb#L4-L14

---

Reply:
> Return the different fields
> - Add the following fields to the response fields
>     - api.last_modified_at: record.registered_at
>     - api.registered_at: record.registered_at

@kou 
In this case, I expect it to be displayed on the screen as follows. What do you think of it?

| Before | After |
|---|---|
|<img width=100% alt="Screenshot 2023-12-26 at 17 58 13" src="https://github.com/clear-code/redmine_full_text_search/assets/45173523/2e69c4ad-e3d0-4992-a1f7-a3f03f84c602">|<img width=100% alt="Screenshot 2023-12-26 at 17 57 27" src="https://github.com/clear-code/redmine_full_text_search/assets/45173523/ba4a30e4-dc24-409e-a096-c7b0de1868f9">


```diff
diff --git a/app/views/search/index.api.rsb b/app/views/search/index.api.rsb
index e61693c..40943cc 100644
--- a/app/views/search/index.api.rsb
+++ b/app/views/search/index.api.rsb
@@ -3,13 +3,15 @@ api.array :results, api_meta(:total_count => @result_set.n_hits,
                              :limit => @search_request.limit) do
   @result_set.each do |record|
     api.result do
-      api.id          record.event_id
-      api.title       record.event_highlighted_title
-      api.type        record.event_type
-      api.url         url_for(record.event_url(:only_path => false))
-      api.description record.event_content_snippets.join("\n")
-      api.datetime    record.event_datetime
-      api.rank        record.rank
+      api.id               record.event_id
+      api.title            record.event_highlighted_title
+      api.type             record.event_type
+      api.url              url_for(record.event_url(:only_path => false))
+      api.description      record.event_content_snippets.join("\n")
+      api.datetime         record.event_datetime
+      api.last_modified_at record.last_modified_at
+      api.registered_at    record.registered_at
+      api.rank             record.rank
     end
   end
 end
diff --git a/app/views/search/index.html.erb b/app/views/search/index.html.erb
index b9a46b3..3071fbe 100644
--- a/app/views/search/index.html.erb
+++ b/app/views/search/index.html.erb
@@ -189,7 +189,13 @@
               <li><%= snippet %></li>
             <% end %>
           </ol>
-        <span class="author"><%= format_time(e.event_datetime) %></span></dd>
+          <span class="last-modified">
+            <%= "#{l(:label_full_text_search_order_target_last_modified_time)}: #{format_time(e.last_modified_at)}" %>
+          </span>
+          <span class="registered">
+            <%= "#{l(:label_full_text_search_order_target_registered_time)}: #{format_time(e.registered_at)}" %>
+          </span>
+        </dd>
       <% end %>
     </dl>
 
(END)
```

Reply:
I just thought `app/views/search/index.html.erb` but outputting in `index.api.erb` too is better. Good catch.

I feel that it's difficult to distinct `last_modified_at` value and `registered_at` value because we don't have any separator between them.

How about using icon instead of `l(:label_full_text_search_order_target_XXX)`?

Reply:
fix: [8ba3f0a](https://github.com/clear-code/redmine_full_text_search/pull/119/commits/8ba3f0a9b7fdc72a8969d34c3000600c1c9e1b2a) Thanks!